yt_dlp/extractor/camfm.py

```diff
@@ -0,0 +1,87 @@
+from re import finditer
+from urllib.parse import urljoin
```

Comment:
```suggestion
import re
import urllib.parse
```

We use from imports only for internal modules for clarity